css/themes/_light.scss

```diff
@@ -91,7 +91,17 @@ $popoverFontColor: #ffffff;
 $popupMenuSelectedItemBackground: rgba(256, 256, 256, .2);
 
 // Toolbar
+$secToolbarBackground: rgba(0, 0, 0, 0.5);
+$sideBarBackground: rgba(0, 0, 0, 0.75);
 $splitterColor: #ccc;
+$toolbarBackground: rgba(0, 0, 0, 0.5);
+$toolbarBadgeBackground: #165ECC;
+$toolbarBadgeColor: #FFFFFF;
+$toolbarButtonColor: #FFFFFF;
+$toolbarSelectBackground: rgba(0, 0, 0, .6);
+$toolbarTitleColor: #FFFFFF;
+$toolbarToggleBackground: #12499C;
```

Comment:
One was created only to change the color scheme, but then I see that we have used the variables file to add colours as well, so currently it's a mess.